src/lib/bootloader/grub2base.rb

```diff
@@ -1,51 +1,59 @@
 # encoding: utf-8
 require "yast"
+require "bootloader/sections"
 require "bootloader/grub2pwd"
 require "bootloader/udev_mapping"
-
-module Yast
+require "bootloader/serial_console"
+require "config_files/grub2/default"
+require "config_files/grub2/grub_cfg"
+require "config_files/matcher"
+require "config_files/placer"
+
+Yast.import "Arch"
+Yast.import "BootArch"
+Yast.import "BootCommon"
+Yast.import "BootStorage"
+Yast.import "HTML"
+Yast.import "Initrd"
+Yast.import "Kernel"
+Yast.import "Mode"
+Yast.import "Pkg"
+Yast.import "Product"
+Yast.import "ProductFeatures"
+Yast.import "Stage"
+Yast.import "Storage"
+Yast.import "StorageDevices"
+
+module Bootloader
   # Common base for GRUB2 specialized classes
-  class GRUB2Base < Module
+  class GRUB2Base < BootloaderBase
+    include Yast::Logger
+    include Yast::I18n
+
     # @!attribute password
     #    @return [::Bootloader::GRUB2Pwd] stored password configuration object
     attr_reader :password
```

Comment:
For whatever reason my yardoc-0.8.7.6 (and also [rubydoc.org](http://www.rubydoc.info/github/yast/yast-bootloader/master/Yast/GRUB2Base#password-instance_method)) ignores markup like this. I got it to render by removing the `@!attribute` line and outdenting the `@return` line.